indexers/vector/AnnoyIndexer/__init__.py

```diff
@@ -41,11 +41,11 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
         _index.build(self.n_trees)
         return _index
 
-    def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
+    def query(self, query_vectors: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
```

Comment:
do not change the signature, I agree `keys` is not perfect, but there is a big chance that people use named arguments (we need to respect BaseClass interface)

Reply:
Done